packages/models-library/src/models_library/api_schemas_directorv2/dynamic_services.py

```diff
@@ -55,26 +52,7 @@ class DynamicServiceCreate(ServiceDetails):
         default=None,
         description="contains harware information (ex. aws_ec2_instances)",
     )
-
-    class Config:
-        schema_extra: ClassVar[dict[str, Any]] = {
-            "example": {
-                "key": "simcore/services/dynamic/3dviewer",
-                "version": "2.4.5",
-                "user_id": 234,
-                "project_id": "dd1d04d9-d704-4f7e-8f0f-1ca60cc771fe",
-                "node_uuid": "75c7f3f4-18f9-4678-8610-54a2ade78eaa",
-                "basepath": "/x/75c7f3f4-18f9-4678-8610-54a2ade78eaa",
-                "product_name": "osparc",
-                "can_save": True,
-                "service_resources": ServiceResourcesDictHelpers.Config.schema_extra[
-                    "examples"
-                ][0],
-                "wallet_info": WalletInfo.Config.schema_extra["examples"][0],
-                "pricing_info": PricingInfo.Config.schema_extra["examples"][0],
-                "hardware_info": HardwareInfo.Config.schema_extra["examples"][0],
-            }
-        }
+    model_config = ConfigDict()
```

Comment:
missing example

Reply:
`bump-pydantic` has this other "bug": it removes unknown config properties (`schema_extra` has been renamed to `json_schema_extra`).